services/web/server/src/simcore_service_webserver/computation_comp_tasks_listening_task.py

```diff
@@ -171,7 +171,7 @@ async def comp_tasks_listening_task(app: web.Application) -> None:
 
 
 async def setup_comp_tasks_listening_task(app: web.Application):
-    task = asyncio.get_event_loop().create_task(comp_tasks_listening_task(app))
+    task = asyncio.create_task(comp_tasks_listening_task(app))
```

Comment:
I understand it is the other way around. We should use ``create_task`` with coroutines since it is a "higher level function" . SEE: https://docs.python.org/3/library/asyncio-task.html#creating-tasks
@GitHK  please ACK

Reply:
my mind inverted them.